src/Components/fe-login/FeLogin.js

```diff
@@ -0,0 +1,55 @@
+import { LitElement, html, css } from 'lit-element';
+import '@lion/form/lion-form';
+import '@lion/input/lion-input';
+import { localize, LocalizeMixin } from '@lion/localize';
+
+export class FeLogin extends LocalizeMixin(LitElement) {
+  static get localizeNamespaces() {
+    return [
+      { 'fe-login': locale => import(`./translations/${locale}.js`) },
+      ...super.localizeNamespaces,
+    ];
+  }
+
+  static get properties() {
+    return {
+      username: { type: String },
```

Comment:
Don't need this ur not passing any param

Reply:
removed